core/templates/components/common-layout-directives/navigation-bars/top-navigation-bar.component.ts

```diff
@@ -139,6 +146,7 @@ export class TopNavigationBarComponent implements OnInit, OnDestroy {
 
   CLASSROOM_PROMOS_ARE_ENABLED = false;
   LEARNER_GROUPS_FEATURE_IS_ENABLED = false;
+  SHOW_FEEDBACK_UPDATES_IN_PROFILE_PIC_DROP_DOWN = false;
```

Comment:
Avoid naming variables with a verb at a start. Those are generally reserved for functions because they perform actions while variables don't.

Reply:
Done.